packages/server/controllers/guilds.js

```diff
@@ -82,6 +82,41 @@ async function getGuildMembers(guildId) {
   return members;
 }
 
+async function getGuildsForUser(userId) {
```

Comment:
Since the API route for this is located in `api/users.js`, let's move this function to `controllers/users.js` for consistency.